files/values.yml.tftpl

```diff
@@ -134,26 +151,32 @@ additionalFilters: |
 
   [FILTER]
       Name                nest
-      Match_regex         application.*|platform.*|fluentbit.*
+      Match_regex         application.*|application-errors.*|platform.*|fluentbit.*
       Operation           lift
       Nested_under        kubernetes
       Add_prefix          Kube.
 
   [FILTER]
       Name                modify
-      Match_regex         application.*|platform.*|fluentbit.*
+      Match_regex         application.*|application-errors.*|platform.*|fluentbit.*
       Remove              Kube.<Metadata_1>
       Remove              Kube.<Metadata_2>
       Remove              Kube.<Metadata_3>
 
   [FILTER]
       Name                nest
-      Match_regex         application.*|platform.*|fluentbit.*
+      Match_regex         application.*|application-errors.*|platform.*|fluentbit.*
       Operation           nest
       Wildcard            Kube.*
       Nested_under        kubernetes
       Remove_prefix       Kube.
 
+  # Retrive logs with status code 4xx and 5xx
+  [FILTER]
+      Name                grep
+      Match               application-errors.*
+      Regex               $log_processed['level'] [45][0-9]{2}
```

Comment:
```suggestion
      Regex               $$log_processed['status'] [45][0-9]{2}
```

I think that you need to escape the `$` character because this file is processed by the `templatefile` terraform function.

The key of the object, as I can see in the logs, is `status`. Can you please check which one is the right one?

Reply:
Yes correct, I checked again on Cloudwatch and the field is `status`